corehq/apps/user_importer/tests/test_importer.py

```diff
@@ -754,3 +755,47 @@ def test_weak_password(self):
             mock.MagicMock()
         )['messages']['rows']
         self.assertEqual(rows[0]['flag'], 'Password is not strong enough. Try making your password more complex.')
+
+
+class TestUserUploadRecord(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        delete_all_users()
+        cls.domain_name = 'mydomain'
+        cls.domain = Domain.get_or_create_with_name(name=cls.domain_name)
+        cls.spec = {
+            'username': 'hello',
+            'name': 'Another One',
+            'password': 123,
+        }
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.domain.delete()
+        UserUploadRecord.objects.all().delete()
+        super().tearDownClass()
+
+    def tearDown(self):
+        Invitation.objects.all().delete()
```

Comment:
Where are invitations created in this test?

Reply:
They are not. I will remove this.